corehq/apps/notifications/ui_notify.py

```diff
@@ -101,3 +111,8 @@ def enabled(self, request):
     ends_on=datetime(2018, 7, 1),
     only_visible_for_users_created_before=datetime(2018, 5, 25),
 )
+
+ABILITY_TO_HIDE_TRANSLATIONS = StaticUINotify(
+    'ability_to_hide_translations',
+    custom_checks=[ensure_app_with_multiple_langs]
```

Comment:
I think it'd be simpler to just not show the bubble in the template if the app doesn't have multiple langs, rather than add complexity to this file. Then you could also avoid attaching the app to the request in `form_source`, which feels a little messy (probably because adding things to the request feels similar to using global variables).

Reply:
That is definitely simpler. Though i liked how all the logic related to the notification stays here with the notification. By having a check in the view separately, breaks up the logic at two different places. But if this looks like overkill, happy to keep it simple as well.

> app to the request

i thought this would be similar to how we have `couch_user` or `domain` on the request object.

Reply:
Yeah, I don't like attaching `couch_user` and `domain` to the request either ;)  I think our bar for adding more data to the request object should be high, especially when the data is as large as an app.